### DISCUSSION TITLE
Champion for PEP 582

### DISCUSSION THREAD

**Discussion by Saphyel** [pdm-project/pdm#1291] [General]
At the moment [PEP 582](https://peps.python.org/pep-0582/) is abandoned, so is anyone interested in lead that PEP ??

If not it's very likely that never gets approved.

**Comment by pawamoy** [comment]
It will not be approved as it is right now indeed. There are multiple points that need to be specified, for example the directory structure of `__pypackages__`. I personally don't feel like I have the experience to champion that PEP :confused:

**Reply by Saphyel** [reply]
You should try! This could be your first PEP and you have half work done and experience for the next PEP and contributions for python!
